pcsx2/Recording/Utilities/InputRecordingLogger.h

```diff
@@ -28,7 +28,31 @@
 
 namespace inputRec
 {
-	void log(const std::string& log);
-	void consoleLog(const std::string& log);
-	void consoleMultiLog(const std::vector<std::string>& logs);
+	static void log(const std::string log)
```

Comment:
This is reverting a commit on master.

Reply:
Can you please link the commit? Tellow has done this rebase and not me, so I'm not sure about what was changed during the rebase unfortunately

Reply:
Here's the PR that it's reverting: https://github.com/PCSX2/pcsx2/pull/4353

Reply:
Oops
I saw that in the merge conflict and thought the change had been in the other direction